docker/build-tools/Dockerfile

```diff
@@ -118,11 +172,24 @@ ENV GO111MODULE=on
 ENV GOPROXY=https://proxy.golang.org
 ENV GOPATH=/go
 ENV GOCACHE=/gocache
-ENV PATH /go/bin:$PATH
+ENV PATH=/go/bin:$PATH
+ENV HOME=/root
```

Comment:
This isn't for Go's benefit, right? Maybe it should be in a separate section of env vars?

Reply:
ya - the golang context copies to /go/bin - so the PATH needs to be set appropriately.